starboard/build/toolchain/BUILD.gn

```diff
@@ -14,7 +14,21 @@
 
 import("//build/config/clang/clang.gni")
 import("//build/toolchain/gcc_toolchain.gni")
+import("//starboard/raspi/shared/toolchain/raspi_shared_toolchain.gni")
```

Comment:
this should ideally go,

same file will have clang3-6 , clang 6-3 , all of these cant be included here
might be good to define a variable like yavor mentioned :  native_ld  which is then set for all 
platforms

Reply:
maybe in the paltform's configuration.gni and then gets imported here

Reply:
this  file shouldnt know about clang3-6, raspi, ps4/5 or any other platform specific gni at this level i think